src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: $transition-ms;
```

don't use it like that, because it's the same as transition: all 0.3s; or transition-duration: 0.3s;
It's too hard for the browser
```suggestion
    transition: transform $transition-ms, color $transition-ms;
```